frontend/jupyter/src/app/pages/form/form-new/volume/existing/existing-volume.component.html

```diff
@@ -5,6 +5,7 @@
     <mat-option [value]="EXISTING_VOLUME_TYPE.PVC" i18n>
       Kubernetes Volume
     </mat-option>
+
```

Comment:
Why?...

Reply:
Upstream added that new line. Came with the copy paste, and when reapplying our custom changes, it didnt get rid of it since the comment operator is at the beginning and end of this block